embd.go

```diff
@@ -176,12 +194,12 @@ package {{.Pkg}}
 {{range .Files}}
 // {{.VarName}} contains contents of "{{.Path}}" file.
 var {{.VarName}} = []byte("{{range .DataFragments}}{{.}}{{end}}")
-{{end}}`[1:]
+{{end}}
 
-/*
-var dirTemplate = `
-var {NAME} = map[string][]byte{
-{ENTRIES}
+{{range $index, $element := .Dirs}}
+var Dir_{{$index}} = map[string][]byte{
```

Comment:
I think I should use [format.Source](https://golang.org/pkg/go/format/#Source) for gofmting. It keeps template more readable and could look like
```go
	w := bytes.Buffer{}
	err = template.Must(template.New("Contents").Parse(Template)).
		Execute(&w, contents)
	if err != nil {
		return err
	}

	src, err := format.Source(w.Bytes())
	if err != nil {
		return err
	}

	err = ioutil.WriteFile(*out, src, 0644)
	if err != nil {
		return err
	}
```
What do you think?